press/api/monitoring.py

```diff
@@ -112,14 +112,14 @@ def targets(token=None):
 def alert(*args, **kwargs):
 	user = frappe.session.user
 	try:
-		monitor_token = frappe.db.get_single_value("Press Settings", "monitor_token", cache=True)
+		webhook_token = frappe.db.get_single_value("Press Settings", "webhook_token", cache=True)
 
-		if frappe.request.args.get("monitor_token") != monitor_token:
+		if frappe.request.args.get("webhook_token") != webhook_token:
 			raise frappe.AuthenticationError("Invalid credentials")
 
-		monitor_token = frappe.db.get_single_value("Press Settings", "monitor_token", cache=True)
+		webhook_token = frappe.db.get_single_value("Press Settings", "webhook_token", cache=True)
 
-		if frappe.request.args.get("monitor_token") != monitor_token:
+		if frappe.request.args.get("webhook_token") != webhook_token:
 			raise Exception
```

Comment:
this is wrong? there is no field `webhook_token` in press settings

and not to mention there are 2 checks for the same thing raising different exceptions

we can just get the monitor server from press settings and then get it's token from it's document? - or store it redundantly in press settings (as it is currently in the form of `monitor_token`) - but add like a fetch on save from the actual monitor doc so that we dont have to add it manually (and also make it read only)

Reply:
ah thanks